pkg/rc/rcmetrics/metrics.go

```diff
@@ -0,0 +1,41 @@
+package rcmetrics
+
+import (
+	"time"
+
+	"github.com/square/p2/Godeps/_workspace/src/github.com/rcrowley/go-metrics"
+
+	"github.com/square/p2/pkg/logging"
+)
+
+const (
+	// Represents the amount of time it takes to process an RC list update
+	// in the farm. Specifically, this means spinning off a goroutine for
+	// each unclaimed RC received from a watch.
+	RCProcessingTimeMetric = "rc_processing_time"
+)
+
+type Metrics struct {
+	Registry metrics.Registry
+	Logger   logging.Logger
+}
+
+func (m *Metrics) SetRegistry(registry metrics.Registry) error {
+	m.Registry = registry
+	return m.Registry.Register(RCProcessingTimeMetric, metrics.NewHistogram(metrics.NewExpDecaySample(1028, 0.015)))
```

Comment:
This went away in a future PR, but it seems like those are just the standard values for a histogram using an exponential decay sample. This is based on docs and also our internal metrics system has these hardcoded